### PR TITLE
presumed smallish corrections for the documentation smoe/docs_translations_mix_01

### PR DESCRIPTION
**Pull request by smoe**
@hansu ?
Thank you tons!
Steffen

### CODE REVIEW

File: docs/src/gui/qtvcp.adoc [hunk 1]
```diff
@@ -277,12 +277,12 @@ Here we show how to add new functions and override existing ones.
...
 import types
 
-# import the handlerfile to get reference to it's libraries.
+# import the handlerfile to get reference to its libraries.
 # use <screenname>_handler
 import qtdragon_handler as hdlr
 
 # This is actually an unbounded function with 'obj' as a parameter.
-# You call this function without the usual preceding 'self.'
+# You call this function without the usual preceding 'self'.
```
Comment (c-morley) [inline docs/src/gui/qtvcp.adoc:285]: 'self.' Is correct

Comment (smoe) [reply docs/src/gui/qtvcp.adoc:285]: Ah. I had obviously misinterpreted that period. May I suggest to change it to 
```
'self.'.
```
?

Comment (c-morley) [reply docs/src/gui/qtvcp.adoc:285]: Ya I almost issued it too :) . Sure that looks good to me. Thanks

Comment (c-morley) [reply docs/src/gui/qtvcp.adoc:285]: Almost missed it I mean.

File: docs/src/gui/qtvcp.adoc [hunk 2]
```diff
@@ -277,12 +277,12 @@ Here we show how to add new functions and override existing ones.
...
 import types
 
-# import the handlerfile to get reference to it's libraries.
+# import the handlerfile to get reference to its libraries.
 # use <screenname>_handler
 import qtdragon_handler as hdlr
 
 # This is actually an unbounded function with 'obj' as a parameter.
-# You call this function without the usual preceding 'self.'
+# You call this function without the usual preceding 'self.'.
```
Comment (petterreinholdtsen) [inline docs/src/gui/qtvcp.adoc:285]: I suspect this should read "'self'.", without the extra dot.

**Comment by petterreinholdtsen** [comment]
The changes look good to me, except one minor question.

File: docs/src/gui/qtvcp.adoc (hunk 2 above)
Comment (smoe) [reply docs/src/gui/qtvcp.adoc:285]: ;)

The final dot is the end of that sentence. The dot embraced by the ''s is meant to reference the use of self with an attribute name, as in 'self.whatever', see https://github.com/LinuxCNC/linuxcnc/pull/2797#discussion_r1430644909 .

Comment (hansu) [reply docs/src/gui/qtvcp.adoc:285]: I tend to the version with the dot at the end as it marks the end of the sentence.

**Pull request merged**